bin/oneoff/fix_level_group_data

```diff
@@ -54,61 +54,55 @@ def get_data(raw_data)
 end
 
 # Retrieves the UserLevel and the (JSON-parsed) LevelSource.data associated with
-# a user_level_id. Raises an exception if the UserLevel or LevelSource is
-# missing.
+# a user_level_id. Raises an exception if the UserLevel is missing.
 # @param user_level_id [Integer] The ID of the user_level.
-# @return [Array[UserLevel, Hash]] An array of a UserLevel and the corresponding
-#   (JSON-parsed) LevelSource.data.
+# @return [Array[UserLevel, {nil | Hash}]] An array of a UserLevel and the
+#   corresponding (JSON-parsed) LevelSource.data.
 def get_user_level_and_level_source_data(user_level_id)
   user_level = UserLevel.find_by_id(user_level_id)
-  unless user_level && user_level.level_source_id
-    raise "Malformed UserLevel: #{user_level_id}"
+  unless user_level
+    raise "Missing UserLevel: #{user_level_id}"
   end
 
   level_source = LevelSource.find_by_id(user_level.level_source_id)
-  unless level_source
-    raise "Missing LevelSource: #{user_level.level_source_id} "\
-      "(#{user_level_id})"
```

Comment:
Of course, this happens (at least on staging). Given that this is not unexpected, I'm simply returning `nil` for `level_source_data` and skipping the LevelGroup UserLevel in this case.